velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -221,5 +221,38 @@ TEST_F(ProbabilityTest, invBetaCDF) {
   VELOX_ASSERT_THROW(invBetaCDF(3, 5, 1.1), "p must be in the interval [0, 1]");
 }
 
+TEST_F(ProbabilityTest, laplaceCDF) {
+  const auto laplaceCDF = [&](std::optional<double> location,
+                              std::optional<double> scale,
+                              std::optional<double> x) {
+    return evaluateOnce<double>("laplace_cdf(c0, c1, c2)", location, scale, x);
+  };
+
+  // Test cases
+  EXPECT_DOUBLE_EQ(0.5, laplaceCDF(0.0, 1.0, 0.0).value());
+  EXPECT_DOUBLE_EQ(0.5, laplaceCDF(5.0, 2.0, 5.0).value());
+  EXPECT_DOUBLE_EQ(0.0, laplaceCDF(5.0, 2.0, -kInf).value());
+  // Test NaN inputs
```

Comment:
You can remove the comments with // Test.... The intent of the comment can be easily inferred from the code.